src/settings_gui.cpp

```diff
@@ -140,6 +142,52 @@ void ShowBaseSetTextfileWindow(TextfileType file_type, const TBaseSet* baseset,
 	new BaseSetTextfileWindow<TBaseSet>(file_type, baseset, content_type);
 }
 
+std::vector<int> refresh_rates = { 30, 60, 75, 90, 100, 120, 144, 240 };
+
+// We want 60Hz as the default for this pointer, though it gets overriden with whatever is in the config.
+int *current_refresh_rate = &refresh_rates[1];
+
+std::string hertz = "Hz";
+std::string current_refresh_rate_str = std::to_string(*current_refresh_rate) + hertz;
```

Comment:
globals should start with a `_`, and constants should be CAPITALS.

Reply:
And constants should also be `static const`